gulpfile.js

```diff
@@ -6,6 +6,8 @@ var ON_DEATH = require('death')({ debug: false });
 var watch = require("gulp-watch");
 var del = require('del');
 var sass = require('gulp-sass');
+var shell = require('shelljs');
+var eachRow = require('gulp-each-row');
```

Comment:
What is the value of this library? Looks like you're just doing a for loop.

Reply:
Is it to signal to gulp when the looping is complete?

Reply:
Was allowing me to to loop the JSON angular-components data, without turning each source dir into a stream, I just needed their names. Was also aiming to get multiple builds running in parallel and deal with them completing asynchronously - have since simplified some (the shell commands are now synchronous) so I could probably eliminate it and go back to standard for loop OR we can retain and I can switch process to return async done functions. I doubt this is a huge priority so suspect we should KISS and got to a for loop.